scripts/build-json-examples.ts

```diff
@@ -33,6 +34,10 @@ const examplesToConvert: Example[] = [
     filename: 'validListedBuildingConsent',
     data: validListedBuildingConsent,
   },
+  {
+    filename: 'priorAproval-largerExtension',
+    data: priorApprovalLargerExtension,
```

Comment:
nit: totally not introduced here, but as we add more examples we should have a think about naming conventions! Especially for application "subtypes" like this one. Dropping `valid` prefix from all seems right as we don't actually intend to publish "invalid" examples.